functable.c

```diff
@@ -10,6 +10,11 @@
 
 #include "gzendian.h"
 
+#ifdef DYNAMIC_CRC_TABLE
```

Comment:
I think this should move down to the crc32-area (although that area seems to be missing a comment to make it more visible) for example after the definiton of crc32_generic, around line 39.